graphql/src/schemas/query.ts

```diff
@@ -187,5 +187,9 @@ export const querySchema = `
       rootId: ID
       leavesOnly: Boolean
      ): [OntologyTree]
+
+    ontologyWords(
+      id: [ID!]
```

Comment:
Other filters that accept multiple values are in plural, like `administrativeDivisionIds`, so it would be nicer if this was too.

Reply:
changed to ids